Lib/test/test_compile.py

```diff
@@ -906,7 +906,7 @@ def load_attr():
                 o.
                 a
             )
-        load_attr_lines = [ 2, 3, 1 ]
```

Comment:
So it seems like we're now generating tracing events for "line zero" of functions, etc.

Is this just because we're tracing function entries in an opcode that's not really associated with any source line, or is there some reason why this is useful for tooling?

Reply:
When generating "call" events, we need a line number.
Previous `f_lasti` would be -1 at function entry, so the line number would be set to `co_firstlineno`.
Now `f_lasti` is 0 or more, so we need an entry in the line number table for it.

`RESUME` instructions never generate "line" events.